### PR TITLE
Krux refactoring

### PR DESCRIPTION
**Pull request by leggsimon**

### CODE REVIEW

File: server/v1/controllers/content.js [hunk 1]
```diff
@@ -10,27 +11,19 @@ module.exports = (req, res) => {
       uuid
     })
     .then((article) => {
-      const metadata = article.metadata
+      const metadata = article.metadata;
+      let result = {};
+      result = {
```
Comment (adgad) [inline server/v1/controllers/content.js]: Maybe dfp and krux should behave the same way?

e.g. 
```
let result = {};
Object.assign(result, getDfpData(metadata, referrer), getKruxData(metadata));

Comment (leggsimon) [inline server/v1/controllers/content.js]: As discussed I am changing the response to be config agnostic eg.
```js
{
  dfpSite: 'String',
  dfpZone: 'String',
  metadata: {
    primarySection: [{id:prefLabel}, {id:prefLabel],
    organisations: [{id:prefLabel}, {id:prefLabel],
    topics: [{id:prefLabel}, {id:prefLabel],
    authors: [{id:prefLabel}, {id:prefLabel],
    genres: [{id:prefLabel}, {id:prefLabel]
  }
}
```

File: server/v1/controllers/content.js [hunk 2]
```diff
@@ -10,27 +11,22 @@ module.exports = (req, res) => {
       uuid
     })
     .then((article) => {
-      const metadata = article.metadata
+      const metadata = article.metadata;
+      let result = {
```
Comment (adgad) [inline server/v1/controllers/content.js]: this can be a const

File: server/v1/controllers/content.js [hunk 3]
```diff
@@ -10,27 +11,22 @@ module.exports = (req, res) => {
...
+        dfpSite: getDfp(metadata, req.query.referrer).dfpSite,
```
Comment (adgad) [inline server/v1/controllers/content.js]: you should call getDfp outside here, so it doesn't get executed twice

**Comment by VladDubrovskis** [comment]
there are two console.logs in the getDfp.js file

**Comment by VladDubrovskis** [comment]
:+1:

**Pull request merged**